File: Neon_Number.py
print("************************")
print("Chack The Neon Number ")
print("************************ \n")

def neon ():
    num = int(input('Enter The Number = '))
    ans = pow(num,2)
    number = 0
    while ans>0:
        number = number + ans%10
        ans=ans//10
    if number==num:
        print('This Is Neon Number = ',num)    
    else:
        print('This Is Normal Number = ',num)
neon()        

File: Neon_Number.php
<?php

function neon($num){
  $ans = $num*$num;
  $res=0;
  while($ans>0){
    $res = $res + ($ans%10);
    $ans = intval($ans/10);
  }
  if($num==$res){
    echo "This is Neon Number = ".$num;
  }
  else{
    echo "This is Normal Number = ".$num;
  }
}
neon(9);

?>